site/diagrams/sequenceDiagrams/tenant_updateTenant_Success.txt

```diff
@@ -1,6 +1,6 @@
 title Update Tenant (success)
 
-Client->+Tenant Service: transfer("tenant", "update", tenant-id, tenantMetaData)
+Client->+Tenant Service: transfer("tenant", "update", ${tenant_id}, tenantMetaData)
```

Comment:
To make it consistent: please change it to "${tenant_id}"
(as you did for remove already ;-))

Reply:
fixed :-)

Reply:
Can you please update the image as well?
Thanks!

Reply:
Should already be up to date :-)

Reply:
Yes, looks good - seems to be a browser cache issue that rendered the "old" image :-)